### PR TITLE
MAINT: interpolate: clean up in fitpack.py

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

File: scipy/interpolate/fitpack.py [hunk 1]
```diff
@@ -50,72 +49,79 @@ def _intc_overflow(x, msg=None):
...
-    The spline has a residual sum of squares fp such that abs(fp-s)/s<=0.001""",None],
-               -1:["""\
-    The spline is an interpolating spline (fp=0)""",None],
-               -2:["""\
+_iermess = {0: ["""\
+    The spline has a residual sum of squares fp such that """
+                """abs(fp-s)/s<=0.001""", None],
+            -1: ["""\
+    The spline is an interpolating spline (fp=0)""", None],
+            -2: ["""\
```
Comment (rgommers) [inline scipy/interpolate/fitpack.py]: While you're cleaning these: ``["""\`` should really just be ``["`` and the string should start on the same line as ``["``.

Also the indent of -1 and -2 still doesn't line up with that of 0.

**Comment by rgommers** [comment]
looks good to me, only the ``"""\`` comment from me

**Comment by WarrenWeckesser** [comment]
> While you're cleaning these: ["""\ should really just be [" and the string should start on the same line as [".

Many of the messages are multiline, so I stuck with the triple quotes on those that didn't really need them for consistency.  I'll take another look; I worked from the bottom up, so when I got to this, my threshold for "good enough" was pretty low. :)

> Also the indent of -1 and -2 still doesn't line up with that of 0.

Actually, they are aligned, otherwise `pep8` (run without the exceptions in `tox.ini`) would have complained.  Maybe an optical illusion?

**Comment by rgommers** [comment]
They do indeed line up. I had to fetch your branch and open it in vim to be sure - good optical illusion. Sorry for the noise.

**Comment by rgommers** [comment]
Multiline string: only makes sense usually if dedented all the way (and then they're ugly). Now the spaces at the start are a bit odd. Try:

    In [1]: from scipy.interpolate.fitpack import _iermess
    In [2]: import warnings
    In [3]: warnings.warn(_iermess[0][0])

On the other hand, otherwise you have to add ``\n`` which is also not pretty. So maybe not worth the bikeshedding.

**Comment by WarrenWeckesser** [comment]
This doesn't look too bad (one of the newlines is added to the beginning of a string instead of the end of the previous string to keep the lines under 80 characters):

```
_iermess = {
    0: ["The spline has a residual sum of squares fp such that "
        "abs(fp-s)/s<=0.001", None],
    -1: ["The spline is an interpolating spline (fp=0)", None],
    -2: ["The spline is weighted least-squares polynomial of degree k.\n"
         "fp gives the upper bound fp0 for the smoothing factor s", None],
    1: ["The required storage space exceeds the available storage space.\n"
        "Probable causes: data (x,y) size is too small or smoothing parameter"
        "\ns is too small (fp>s).", ValueError],
    2: ["A theoretically impossible results when finding a smoothin spline\n"
        "with fp = s. Probably causes: s too small. (abs(fp-s)/s>0.001)",
        ValueError],
    3: ["The maximal number of iterations (20) allowed for finding smoothing\n"
        " spline with fp=s has been reached. Probably causes: s too small.\n"
        "(abs(fp-s)/s>0.001)", ValueError],
    10: ["Error on input data", ValueError],
    'unknown': ["An error occurred", TypeError]
}
```

**Comment by argriffing** [comment]
> Probably causes: s too small.

spelling

**Comment by rgommers** [comment]
@WarrenWeckesser thanks, looks much better now. Merging.

**Pull request merged**